docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -20,28 +20,39 @@ For example, version 7.0.203 ships with .NET 7, is the second minor Visual Studi
 
 The support timeframe for the SDK typically matches that of the Visual Studio version it's included in.
 
-| SDK Version      | MSBuild/Visual Studio version | Ship date    | Lifecycle |
-|------------------|--------------------|--------------|-----------|
-| 2.1.5xx          | 15.9               | Nov '18      | Aug '21   |
-| 2.1.8xx          | 16.2 (No VS)       | July '19     | Aug '21   |
-| 3.1.1xx          | 16.4               | Dec '19      | Oct '21   |
-| 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
-| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
-| 5.0.2xx          | 16.9               | March '21    | May '22   |
-| 5.0.3xx          | 16.10              | May '21      | Aug '21   |
-| 5.0.4xx          | 16.11              | Aug '21      | May '22   |
-| 6.0.1xx          | 17.0               | Nov '21      | Nov '24<sup>1</sup>   |
-| 6.0.2xx          | 17.1               | Feb '22      | May '22   |
-| 6.0.3xx          | 17.2<sup>3</sup>   | May '22      | Oct '23   |
-| 6.0.4xx          | 17.3               | Aug '22      | Nov '24<sup>2</sup>   |
-| 7.0.1xx          | 17.4               | Nov '22      | May '24   |
-| 7.0.2xx          | 17.5<sup>3</sup>   | Feb '23      | May '23   |
-| 7.0.3xx          | 17.6               | May '23      | May '24   |
-| 7.0.4xx          | 17.7               | Aug '23      | May '24   |
-| 8.0.1xx          | 17.8               | Nov '23      | Nov '25<sup>1</sup>    |
-| 8.0.2xx          | 17.9<sup>3</sup>   | Feb '24      | May '24   |
-| 8.0.3xx          | 17.10              | May '24      | TBD       |
-| 8.0.4xx          | 17.11              | Aug '24      | Nov '25<sup>2</sup>       |
+<!-- markdownlint-disable MD033 -->
+<details>
+<summary>Out of support .NET versions</summary>
```

Comment:
```suggestion
<summary>Expand for out-of-support .NET versions</summary>
```

Suggesting this so readers don't think this expander is a title for the always-visible table:

<img width="456" alt="image" src="https://github.com/user-attachments/assets/4ae31291-7acd-4caa-97a3-c30d5771a8d4">